CDCgov/prime-reportstream
# Update FHIRFunctions to base64 encode messages going to poison queue

This PR updates FHIRFunctions to base64 encode poison queue messages in order to fix #16609.

Test Steps:
1. Follow repro steps in ticket but now see that the poison queue message does actually trigger the translate step to run

## Changes
- FHIRFunctions now base64 encoded messages being put in the poison queue
- Tests updated to expect base64 encoded messages 

### Testing

## Linked Issues
- Fixes #16609
